sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -159,6 +262,97 @@ string_class ProgramManager::getProgramBuildLog(const RT::PiProgram &Program) {
   return Log;
 }
 
+struct ImageDeleter {
+  void operator()(DeviceImage *I) {
+    delete[] I->BinaryStart;
+    delete I;
+  }
+};
+
+ProgramManager::ProgramManager() {
+  const char *SpvFileC = std::getenv(UseSpvEnv.c_str());
+  // If a SPIRV file is specified with an environment variable,
+  // register the corresponding image
+  if (SpvFileC)
+    SpvFile = SpvFileC;
+  if (!SpvFile.empty()) {
+    // The env var requests that the program is loaded from a SPIRV file on disk
+    std::ifstream File(SpvFile, std::ios::binary);
+
+    if (!File.is_open())
+      throw runtime_error(std::string("Can't open file specified via ") +
```

Comment:
```suggestion
      throw sycl::runtime_error(std::string("Can't open file specified via ") +
```

Reply:
Why? I don't see the problem here.

Reply:
I suggest this change everywhere because personally I think it's confusing for reader which exception is thrown here, sycl one or std. I'm OK if you ignore this suggestion.

Reply:
Personally, I think that this distinction is fairly obvious here, since std namespace is specified extensively in this source file, but I'm not against the change. It would make more sense as a separate commit though, since this is not the only use of `runtime_error` in the file.